### ISSUE TITLE
Changing style set is not reflected in the Figma

### ISSUE THREAD

**Issue by zwierzyniec** [six7/figma-tokens#191]
Figma Desktop App version 99.0

Token set feature does not seem to work. As you can see on the screenshots below, btn color is set to `$primary` and when I switch to a different set the same token `$primary` should have value of `#000000` . As you can see change is reflected in the inspector but not in the frontend.

"main" token set:
![image](https://user-images.githubusercontent.com/54911307/128329335-8b0723ca-a14b-4808-bf3c-c9133646b68c.png)
![image](https://user-images.githubusercontent.com/54911307/128329265-6e19401e-5de4-4f71-a138-799dc8eb4ffb.png)

"dark" token set:
![image](https://user-images.githubusercontent.com/54911307/128329429-69068de5-3ffb-4843-b452-7c978e3b144c.png)
![image](https://user-images.githubusercontent.com/54911307/128329391-56ba4f48-0c27-46a5-bc84-aa00a0adc84a.png)

**Comment by six7** [comment]
Did you hit Update after you made any changes? The recent version changed that you have to hit Update whenever you made a theme switch, although I'm thinking of reverting that according to #181

**Comment by zwierzyniec** [comment]
Yes, I did. But I didn't notice that file is just crushing after that. 
Now it seems that after the update confirmation toaster appears the file is not responding (reproducible issue). When I try to
I tested it on a larger file with my design system. And now after launching the token plugin and clicking through the token sets the file crashes again.

**Comment by six7** [comment]
Could you provide a screen recording of the error, or possibly a File you could share? That way I could debug properly 👍
